storage/scylladb_client.h

```diff
@@ -14,6 +14,8 @@ extern "C" {
 #include "scylladb_utils.h"
 
 #define DB_UUID_STRING_LENGTH CASS_UUID_STRING_LENGTH
+
+typedef enum { DB_USAGE_REATTACH = 0, DB_USAGE_NULL, NUM_DB_USAGE } db_client_usage_t;
```

Comment:
I don't realize the meaning of enum value `DB_USAGE_NULL`.

Reply:
`DB_USAGE_NULL` is for non-preserved usage with used-defined keyspace.
Currently, we use it in `test_scylladb`. In the beginning, I use an enum value `DB_USAGE_TEST` for testing. But if run several tests(tsan, asan...) in the same keyspace, the tests might fail due to one test delete table before other tests finish.

Therefore, we set `DB_USAGE_NULL` to db_client and give different keyspace names for each test.

I will add descriptions for `DB_USAGE_NULL` in the source code.